Lunch/AppDelegate.swift

```diff
@@ -16,6 +19,18 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
         // Override point for customization after application launch.
+        Parse.enableLocalDatastore()
+        let filePath = NSBundle.mainBundle().pathForResource("Keys", ofType: "plist")
```

Comment:
If you want to use force-unwrapping, it's better to put it at the end of the line here instead of everywhere you use it. `let filePath = NSBundle.mainBundle().pathForResource("Keys", ofType: "plist")!` This is true everywhere you're doing it, I won't leave comments on the other places